ev3/ev3dev.py

```diff
@@ -208,15 +208,15 @@ def __init__(self, port=-1, name=None):
         sensor_existing = False
         if (port > 0):
             self.port = port
-            for p in glob.glob('/sys/class/lego-sensor/sensor*/port_name'):
+            for p in glob.glob('/sys/class/msensor/sensor*/port_name'):
```

Comment:
The class name is change to lego-sensor in the new kernel. Please see http://www.ev3dev.org/docs/drivers/lego-sensor-class/
Which kernel do you use?